recipes/qsimcirq/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "qsimcirq" %}
+{% set version = "0.19.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/quantumlib/qsim/archive/refs/tags/v0.19.0.tar.gz
+  sha256: 81101fe9e6356e54125f703956e787aa366717c1e6e77aea794879b53c84ff60
+
+build:
+  skip: true  # [py2k]
+  skip: true  # [win]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pybind11-abi
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - openmpi
+    - cmake >=3.27
+  host:
+    - python
+    - pybind11
+    - make
+    - setuptools >=45
+    - wheel
+    - pip
+  run:
+    - python
+    - pybind11
+    - absl-py
+    - cirq-core
+    - typing_extensions
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - qsimcirq
+  source_files:
+    - qsimcirq_tests
+  commands:
+    - pip check
+    - cd qsimcirq_tests && pytest -vvv .
+  requires:
+    - pip
+    - pytest
+
+about:
+  home: https://quantumai.google/qsim
+  summary: Quantum circuit simulators qsim and qsimh.
+  license: Apache-2.0
+  license_file:
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - basnijholt
+    - BastianZim
```

Comment:
@BastianZim, could you verify here again that you want to be a maintainer (like done here https://github.com/conda-forge/cirq-feedstock/pull/13#issuecomment-1828689789)?

Reply:
Yes, I'm happy to!

Reply:
Thanks @BastianZim!

All builds are passing. Let's get this in ASAP 😄